enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -1,12 +1,10 @@
-{{- $promDomainUnderscored := .Values.prometheusDomain | replace "." "_" }}
-
```

Comment:
I think we need a comment here explaining that this file will be parsed by es-monitor-api. I wonder also if we need to provide a reference to the list of functions so readers understand where e.g. `dot2underscore` comes from. Maybe if we keep those functions in their own file we can just link to it here in a comment?

Reply:
Agree on the note, trying to sort out how to add a note that doesn't also end up in the final prometheus config file haha